### ISSUE TITLE
Tabindex and document role have to be assigned for terminal output div container

### ISSUE THREAD

**Issue by jooyoungseo** [rstudio/rstudio#6926]
### System details

    RStudio Edition : Server
    RStudio Version : 1.3.959
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem

1. Enable "Help / Accessibility / Screen Reader Support".
2. Move focus to terminal by pressing Alt+Shift+T on Windows or Linux; "⌥⇧T" on Mac.
3. Type `ls -1` in terminal.
4. Try to find a way to review the announced output in detail using arrows or tab keys.

### Describe the problem in detail

The terminal output is not focusable for screen reader users for detailed review.

### Describe the behavior you expected

Please implement `tabindex = "0"` for the terminal output div container and assign `"document"` role like "Console Output" area. Keyboard shortcut would be a big plus; however, I have no idea off the top of my head at this moment.

Once focusable, this terminal output region should be included in the F6 pane-cycle as suggested in [#6882](https://github.com/rstudio/rstudio/issues/6882).

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by gtritchie** [comment]
We use the same component for the terminal as VSCode: xterm.js, though a slightly older version. Be good to compare our behavior with theirs.

**Comment by jooyoungseo** [comment]
In VSCode, you have to switch screen readers' virtual cursor in order to review terminal output manually, which is not convenient, either. And, yes, their terminal output is not focusable, which is another pain.

**Comment by gtritchie** [comment]
This work needs to happen in xterm.js. See:

https://github.com/microsoft/vscode/issues/98918
https://github.com/xtermjs/xterm.js/pull/3020

**Comment by gtritchie** [comment]
I'm going to close this. RStudio is using an ancient build of xterm.js but I am going to be updating to a current build (see #9195).

I will reevaluate the terminal's screen reader behaviour once that is complete (and compare it to VSCode's behaviour) and see where we are at!
